opendbc/car/honda/carstate.py

```diff
@@ -131,6 +131,7 @@ def update(self, can_parsers) -> structs.CarState:
     else:
       gear_position = self.shifter_values.get(cp.vl[self.gearbox_msg]["GEAR_SHIFTER"], None)
       ret.gearShifter = self.parse_gear_shifter(gear_position)
+      ret.regenBraking = cp.vl[self.gearbox_msg].get("REGEN_STAGE_SELECTION", 0) > 0
```

Comment:
What is this? Can we be explicit? Now you can silently skip checking regen paddle if someone renames it in DBC

Reply:
Your question is reasonable. I'm trying to resolve #1100 incrementally. It was a blocker for the CAN-FD Accord.

Shortly we'll catch it by shipping #2569 with additions to read the regen paddle state, along the way resolving #1100, and then reverting commaai/openpilot#35929 (nice test BTW).

If I can't do that by tomorrow, I'll make this a normal read gated by `transmissionType == automatic and not CAR.ACURA_RDX:`, that's basically what the above code does.